client/state/site-settings/test/reducer.js

```diff
@@ -0,0 +1,302 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import {
+	SITE_SETTINGS_RECEIVE,
+	SITE_SETTINGS_REQUEST,
+	SITE_SETTINGS_REQUEST_FAILURE,
+	SITE_SETTINGS_REQUEST_SUCCESS,
+	SITE_SETTINGS_SAVE,
+	SITE_SETTINGS_SAVE_FAILURE,
+	SITE_SETTINGS_SAVE_SUCCESS,
+	SITE_SETTINGS_UPDATE,
+	SERIALIZE,
+	DESERIALIZE
+} from 'state/action-types';
+import { items, requesting, saving } from '../reducer';
+
+describe( 'reducer', () => {
+	before( () => {
+		sinon.stub( console, 'warn' );
+	} );
+
+	after( () => {
+		console.warn.restore(); // eslint-disable-line
+	} );
+
+	describe( '#requesting()', () => {
+		it( 'should default to an empty object', () => {
+			const state = requesting( undefined, {} );
+
+			expect( state ).to.eql( {} );
+		} );
+
+		it( 'should set requesting value to true if request in progress', () => {
+			const state = requesting( undefined, {
+				type: SITE_SETTINGS_REQUEST,
+				siteId: 2916284
+			} );
+
+			expect( state ).to.eql( {
+				2916284: true
+			} );
+		} );
+
+		it( 'should accumulate requesting values', () => {
+			const previousState = deepFreeze( {
+				2916284: true
+			}	);
```

Comment:
Tab here should be a space before closing paren.

There are a few more cases in this file. Could be GitHub's preview tricking me (it's been strange with spacing lately).